assets/un_street_map.yml

```diff
@@ -0,0 +1,497 @@
+version: 8
+sprite: https://pro-ags2.dfs.un.org/arcgis/rest/services/Hosted/Unite_StreetMapVT_CVW_V01/VectorTileServer/resources/sprites/sprite_S
+glyphs: https://pro-ags2.dfs.un.org/arcgis/rest/services/Hosted/Unite_StreetMapVT_CVW_V01/VectorTileServer/resources/fonts/{fontstack}/{range}.pbf
+sources:
+  esri:
+    type: vector
+    tiles:
+      - https://pro-ags2.dfs.un.org/arcgis/rest/services/Hosted/Unite_StreetMapVT_CVW_V01/VectorTileServer/tile/{z}/{y}/{x}.pbf
+    minzoom: 0
+    maxzoom: 23
+layers:
+  - !!inc/file layers/background.yml
+  - !!inc/file un_street_map/Landuse/LND_Landcover_S/Forest.yml
```

Comment:
you need to add `hillshade` layer like other styles do. Otherwise, hillshade button in GeoHub does not function.

https://github.com/UNDP-Data/style/blob/d616ed7308cf33c7e997512320d540d927b96758/assets/positron.yml#L26